packages/logger/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "@glif/logger",
+    "version": "2.0.0-alpha.16",
+    "description": "Handles logging for GLIF apps, modules and components",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+        "clean": "rimraf dist/* && mkdirp dist",
+        "build-browser": "cross-env BABEL_ENV=browser ../../node_modules/.bin/babel ./src --out-dir ./lib --extensions \".ts,.tsx\" --source-maps --ignore '**/__tests__'",
+        "build-module": "cross-env BABEL_ENV=module ../../node_modules/.bin/babel ./src --out-dir ./module --extensions \".ts,.tsx\" --source-maps --ignore '**/__tests__'",
```

Comment:
Re your comment, yeah i dont think we'll need these either if we're just running this in nextjs for our purposes. Nextjs can handle the what's currently in the `build-node` script, so we can probably delete these and just keep that